vh_patterns_dataset.py

```diff
@@ -1858,7 +1858,7 @@ def show_implementation_details(id):
             (1,F_M_U_NT):(F_M_U_NT,1),(1,'y'):('y',2),(1,Long_F_H_R_T):(Long_F_H_R_T,2),
             (1,F_M_R_T):(F_M_R_T,2),(1,Long_F_M_R_T):(Long_F_M_R_T,2),(1,'u'):('u',3),
             (1,Long_B_H_R_T):(Long_B_H_R_T,3),(1,'o'):('o',3),
-            (1,Long_B_M_R_T):(Long_B_M_R_T,3),
+            (1,Long_B_M_R_T):(Long_B_M_R_T,3),50
```

Comment:
the 50 here seems out of place

Reply:
Oh, wow. Sorry. That's a "counter" I must have forgotten to delete

Reply:
Fixed it